mnemosyne/core/beam.py

```
@@ -989,6 +989,83 @@ def init_beam(db_path: Path = None):
+            """)
+            cursor.execute("DROP TABLE memory_embeddings")
+            cursor.execute("ALTER TABLE _me_rebuilt RENAME TO memory_embeddings")
+            conn.commit()
+        except Exception:
+            conn.rollback()
+            # Clean up temp table if it was created before the failure
+            try:
+                cursor.execute("DROP TABLE IF EXISTS _me_rebuilt")
+                conn.commit()
+            except Exception:
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)